<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select Type of Injury</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TITLE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1021</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select type of Injury</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1001</width>
      <height>481</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-29</y>
       <width>986</width>
       <height>600</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="burn_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>BURN</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="electric_shock_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ELECTRIC 
 SHOCK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="laceration_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>LACERATION</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="cut_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>CUT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="poison_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>POISONING</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="puncture_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>PUNCTURE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bruises_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BRUISES</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="others_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>OTHERS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="breathing_button">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>190</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>DIFFUCULT 
 BREATHING</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>TITLE</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
